t/io/say.t

```diff
@@ -13,15 +13,12 @@ BEGIN {
 # a print opcode, so it's more or less guaranteed to behave
 # the same way as print in any case.
 
-use strict 'vars';
-use feature "say";
-
-no strict 'refs';
-
 say "1..13";
 
 my $foo = 'STDOUT';
+no strict 'refs';
```

Comment:
could have use a block, but this is fine as it

Reply:
True, but it's useful to have a few places in the test suite where we demonstrate how to switch from the default (`strict` on) to turning part of `strict` off and then back on, all within same block.

I'm working my way through the entire test suite, so I'm trying to avoid a lot of fine-tuning.